stack/templates/job.yaml

```diff
@@ -0,0 +1,179 @@
+{{ $global := . }}
+{{ range $jobName, $jobValues := .Values.jobs }}
+  {{- $globalValuesDict := $global.Values.global | toYaml -}}
+  {{- $values := fromYaml $globalValuesDict -}}
+  {{- $values = set $values "name" $jobName -}}
+  {{- $values := mergeOverwrite $values $jobValues -}}
+  {{- $job := dict "Chart" $global.Chart "Release" $global.Release "Capabilities" $global.Capabilities "Values" $values -}}
+{{- with $job -}}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "service.fullname" . }}
+  labels:
+    {{- include "service.labels" . | nindent 4 }}
+  annotations:
+    {{- toYaml (mergeOverwrite
+                (dict)
+                (fromYaml (include "stack.annotations" $job ))
+      ) | nindent 4 }}
+spec:
+  {{- if .Values.activeDeadlineSeconds }}
+  activeDeadlineSeconds: {{ .Values.activeDeadlineSeconds }}
+  {{- end }}
+  {{- if .Values.ttlSecondsAfterFinished }}
+  ttlSecondsAfterFinished: {{ .Values.ttlSecondsAfterFinished }}
+  {{- end }}
+  {{- if .Values.backoffLimit }}
+  backoffLimit: {{ .Values.backoffLimit }}
+  {{- end }}
+  {{- if .Values.completions }}
+  completions: {{ .Values.completions }}
+  {{- end }}
+  {{- if .Values.parallelism }}
+  parallelism: {{ .Values.parallelism }}
+  {{- end }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
```

Comment:
did you want to add linkerd disabled?

Reply:
ah yeah I just forgot to push the change